configure

```diff
@@ -784,6 +784,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
This part already was added and then removed back because this was a feature from still not released autoconf version. What version do you use? Is it already released?